packages/ingest/lock.js

```diff
@@ -13,8 +13,18 @@ async function delay(t) {
 }
 
 async function countLock(bucket, pName) {
```

Comment:
We should comment at least this function and the new one you wrote with jsdocs.

Reply:
Docs written